components/core/Settings/Jumper.js

```diff
@@ -0,0 +1,394 @@
+import * as React from "react";
+import * as Jumper from "~/components/core/Jumper";
+import * as System from "~/components/system";
+import * as Logging from "~/common/logging";
+import * as Strings from "~/common/strings";
+import * as Styles from "~/common/styles";
+import * as Constants from "~/common/constants";
+import * as SVG from "~/common/svg";
+import * as Validations from "~/common/validations";
+import * as Events from "~/common/custom-events";
+import * as Actions from "~/common/actions";
+import * as UserBehaviors from "~/common/user-behaviors";
+
+import { css } from "@emotion/react";
+import { useSettingsContext } from "~/components/core/Settings/Provider";
+import { AnimatePresence, motion } from "framer-motion";
+
+import ProfilePhoto from "~/components/core/ProfilePhoto";
+
+import { Input } from "~/components/system/components/Input";
+import { Textarea } from "~/components/system/components/Textarea";
+import { ButtonPrimary, ButtonSecondary } from "~/components/system/components/Buttons";
+import * as Type from "~/components/system/components/Typography";
+
+const STYLES_FLEX_ROW = `
```

Comment:
We have a style in common/styles.js for this called HORIZONTAL_CONTAINER. There's other commonly used styles there. Go check it out and see. 

To use it, import it like:
`import * as Styles from "~/common/styles";`
 and use it like this:
![image](https://user-images.githubusercontent.com/33686587/141209817-c5f7bd61-224f-46f1-af24-5a13f52cfd8a.png)

![image](https://user-images.githubusercontent.com/33686587/141209895-c16eac02-a143-4d39-a399-4df121e7dfd0.png)